app/code/community/Bolt/Boltpay/Model/Payment.php

```diff
@@ -1071,4 +1068,95 @@ protected function isPartialRefundFixingMismatch(Mage_Sales_Model_Order $order,
 
         return false;
     }
+
+    /**
+     * Whether this method can accept or deny payment
+     *
+     * @param Mage_Payment_Model_Info $payment
+     *
+     * @return bool
+     */
+    public function canReviewPayment(Mage_Payment_Model_Info $payment)
+    {
+       return $payment->getAdditionalInformation('bolt_transaction_status') == self::TRANSACTION_REJECTED_REVERSIBLE;
+    }
+
+    /**
+     * Attempt to accept a payment that us under review
```

Comment:
that *is*

Reply:
maybe "Attempt to approve a order"? payment is already taken, decision is whether we approve it or not